# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/WebrtcTCPSocketWrapper.h
# Commit: b879a0044cf3
# Full Hash: b879a0044cf39f0230879d075c915485affb7707
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Teach WebrtcTCPSocket to bind to a specific host/port. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45102
# ==============================================================================

diff -r f5353ea9f1df -r b879a0044cf3 media/mtransport/WebrtcTCPSocketWrapper.h
--- a/media/mtransport/WebrtcTCPSocketWrapper.h	Tue Sep 17 18:17:14 2019 +0000
+++ b/media/mtransport/WebrtcTCPSocketWrapper.h	Tue Sep 17 18:17:18 2019 +0000
@@ -42,6 +42,7 @@
   explicit WebrtcTCPSocketWrapper(WebrtcTCPSocketCallback* aCallbacks);
 
   virtual void AsyncOpen(const nsCString& aHost, const int& aPort,
+                         const nsCString& aLocalAddress, const int& aLocalPort,
                          const std::shared_ptr<NrSocketProxyConfig>& aConfig);
   virtual void SendWrite(nsTArray<uint8_t>&& aReadData);
   virtual void Close();